tools/webpack/packages.js

```diff
@@ -111,8 +109,6 @@ module.exports = function (
 			'objectFitPolyfill/dist/objectFitPolyfill.min.js',
 		'wp-polyfill-inert.min.js': 'wicg-inert/dist/inert.min.js',
 		'moment.min.js': 'moment/min/moment.min.js',
-		'react.min.js': 'react/umd/react.production.min.js',
-		'react-dom.min.js': 'react-dom/umd/react-dom.production.min.js',
```

Comment:
Also just like the backported PR, this moves react scripts we used in Core from the deprecated umd builds to CJS.